Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs

```diff
@@ -96,70 +96,63 @@ public bool IsActiveBuildTargetValid()
-            { Configurations.AndroidMinSdkVersion,
-                new ConfigGetter(() => { return PlayerSettings.Android.minSdkVersion >= MinAndroidSdk; }, BuildTarget.Android) },
+            { Configurations.AndroidMultiThreadedRendering, new ConfigGetter(() => !PlayerSettings.GetMobileMTRendering(BuildTargetGroup.Android), BuildTarget.Android) },
+            { Configurations.AndroidMinSdkVersion, new ConfigGetter(() =>  PlayerSettings.Android.minSdkVersion >= MinAndroidSdk, BuildTarget.Android) },
 
             // iOS Settings
-            { Configurations.IOSMinOSVersion, new ConfigGetter(() => {
-                    float version;
-                    if (!float.TryParse(PlayerSettings.iOS.targetOSVersionString, out version)) { return false; }
-                    return version >= iOSMinOsVersion; }, BuildTarget.iOS) },
-            { Configurations.IOSArchitecture, 
-                new ConfigGetter(() => { return PlayerSettings.GetArchitecture(BuildTargetGroup.iOS) == RequirediOSArchitecture; }, BuildTarget.iOS) },
-            { Configurations.IOSCameraUsageDescription, 
-                new ConfigGetter(() => { return !string.IsNullOrWhiteSpace(PlayerSettings.iOS.cameraUsageDescription); }, BuildTarget.iOS) },
+            { Configurations.IOSMinOSVersion, new ConfigGetter(() => float.TryParse(PlayerSettings.iOS.targetOSVersionString, out float version) ? version >= iOSMinOsVersion : false, BuildTarget.iOS) },
+            { Configurations.IOSArchitecture, new ConfigGetter(() => PlayerSettings.GetArchitecture(BuildTargetGroup.iOS) == RequirediOSArchitecture, BuildTarget.iOS) },
+            { Configurations.IOSCameraUsageDescription, new ConfigGetter(() => !string.IsNullOrWhiteSpace(PlayerSettings.iOS.cameraUsageDescription), BuildTarget.iOS) },
         };
 
         // The configure functions for each type of setting
         private static readonly Dictionary<Configurations, Action> ConfigurationSetters = new Dictionary<Configurations, Action>()
         {
-            { Configurations.LatestScriptingRuntime, () => { SetLatestScriptingRuntime(); } },
-            { Configurations.ForceTextSerialization,  () => { SetForceTextSerialization(); } },
-            { Configurations.VisibleMetaFiles,  () => { SetVisibleMetaFiles(); } },
-            { Configurations.VirtualRealitySupported,  () => { XRSettingsUtilities.LegacyXREnabled = true; } },
-            { Configurations.SinglePassInstancing,  () => { MixedRealityOptimizeUtils.SetSinglePassInstanced(); } },
-            { Configurations.SpatialAwarenessLayer,  () => { SetSpatialAwarenessLayer(); } },
+            { Configurations.LatestScriptingRuntime,SetLatestScriptingRuntime },
+            { Configurations.ForceTextSerialization, SetForceTextSerialization },
+            { Configurations.VisibleMetaFiles, SetVisibleMetaFiles },
+            { Configurations.VirtualRealitySupported, () => XRSettingsUtilities.LegacyXREnabled = false },
```

Comment:
Why did this one flip from `true` to `false`? It should be enabling the XR setting

Reply:
Nice catch, clumsy me